sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/RelationshipsSyncSamples.java

```diff
@@ -0,0 +1,176 @@
+package com.azure.digitaltwins.core;
+
+    import com.azure.core.http.policy.HttpLogOptions;
+    import com.azure.core.http.rest.PagedIterable;
+    import com.azure.core.http.rest.Response;
+    import com.azure.core.util.Context;
+    import com.azure.digitaltwins.core.helpers.ConsoleLogger;
+    import com.azure.digitaltwins.core.helpers.SamplesArguments;
+    import com.azure.digitaltwins.core.helpers.SamplesConstants;
+    import com.azure.digitaltwins.core.helpers.UniqueIdHelper;
+    import com.azure.digitaltwins.core.implementation.models.ErrorResponseException;
+    import com.azure.digitaltwins.core.models.*;
+    import com.azure.identity.ClientSecretCredentialBuilder;
+    import com.fasterxml.jackson.core.JsonProcessingException;
+    import com.fasterxml.jackson.databind.ObjectMapper;
+
+    import java.io.IOException;
+    import java.net.HttpURLConnection;
+    import java.util.*;
+    import java.util.function.Function;
+
+public class RelationshipsSyncSamples {
+    private static DigitalTwinsClient client;
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    public static Function<Integer, String> randomIntegerStringGenerator = (maxLength) -> {
+        int randInt = new Random().nextInt((int)Math.pow(10, 8) - 1) + 1;
+        return String.valueOf(randInt);
+    };
+
+    public static void main(String[] args) throws IOException {
+
+        SamplesArguments parsedArguments = new SamplesArguments(args);
+
+        client = new DigitalTwinsClientBuilder()
+            .credential(
+                new ClientSecretCredentialBuilder()
+                    .tenantId(parsedArguments.getTenantId())
+                    .clientId(parsedArguments.getClientId())
+                    .clientSecret(parsedArguments.getClientSecret())
+                    .build()
+            )
+            .endpoint(parsedArguments.getDigitalTwinEndpoint())
+            .httpLogOptions(
+                new HttpLogOptions()
+                    .setLogLevel(parsedArguments.getHttpLogDetailLevel()))
+            .buildClient();
+
+        runRelationshipsSample();
+    }
+
+    @SuppressWarnings("rawtypes")
```

Comment:
What does this do?

Reply:
needs to go

Reply:
it allows us to use Response without any type parameters.